test/widgets/action_sheet_test.dart

```diff
@@ -32,6 +33,17 @@ import 'compose_box_checks.dart';
 import 'dialog_checks.dart';
 import 'test_app.dart';
 
+GetMessagesResult _getMessagesResult(Message message) {
+  return GetMessagesResult(
+    anchor: message.id,
+    foundNewest: true,
+    foundOldest: true,
+    foundAnchor: true,
```

Comment:
Does the test rely on `foundAnchor: true` or `anchor: message.id`? If not, `eg.newestResult` suffices.

Reply:
Yeah, `newestResult` would be an improvement. (Though it doesn't currently live in `eg`; maybe it should, with a longer name in that case.)

Adding a separate prep commit to switch to that.